# Use of /** so that the configurations hold for controller
# managers regardless of their namespace. Usefull in multi-robot setups.
/**/controller_manager:
  ros__parameters:
    update_rate: 200

    # Broadcasters
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    lbr_state_broadcaster:
      type: lbr_ros2_control/LBRStateBroadcaster

    lbr_virtual_ft_broadcaster:
      type: lbr_ros2_control/LBRVirtualFTBroadcaster

    # Controllers
    position_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    forward_position_controller:
      type: position_controllers/JointGroupPositionController

/**/position_trajectory_controller:
  ros__parameters:
    joints:
      - A1
      - A2
      - A3
      - A4
      - A5
      - A6
      - A7
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    state_publish_rate: 50.0
    action_monitor_rate: 20.0

/**/forward_position_controller:
  ros__parameters:
    joints:
      - A1
      - A2
      - A3
      - A4
      - A5
      - A6
      - A7
    interface_name: position
